### DISCUSSION TITLE
Android 手机上使用 clash for android + v2rayng 来体验 Xray 最新功能

### DISCUSSION THREAD

**Discussion by zhangsh-cn** [XTLS/Xray-core#176] [Show and tell]
当然，这个也可以在任意设备上使用 clash + xray。

Clash 优点：可以根据规则来匹配某请求是否走代理。（当然也支持不同的请求走不同的代理，我不使用这个功能）
Xray 优点：太多了。

新的 VMESS AHEAD 和 VLESS 一直不能进入 clash core，所以只能用 Cash + Xray 的方式了。

想问下，有用户这样用吗？性能和耗电量怎么样？

**Comment by zhuzeitou** [comment]
路由规则v2ray/xray本身就有啊，不必再去借助其他工具的吧
v2rayng也有路由设置功能，实在不行手动json肯定能满足需求了
